modules/examples/error-framework/JsExampleErrorRaiser/index.js

```diff
@@ -149,7 +149,7 @@ boot_module(async ({
   mod.provides.example_raiser.raise_error(error);
   check_conditions(mod);
   await sleep(1000);
-  mod.provides.example_raiser.clear_error(error.type, error.sub_type);
+  mod.provides.example_raiser.clear_error(error.type, true);
```

Comment:
See above:
```suggestion
  // Clear all errors that match type of raised error object (In this case: 1)
  mod.provides.example_raiser.clear_error(error.type, true);
```

Reply:
I must have overlooked that when looking at the everestjs code, but I've reverted the commit